.github/workflows/deploy.yml

```diff
@@ -0,0 +1,128 @@
+      TF_CLI_ARGS_apply: ${{ inputs.TF_CLI_ARGS_apply }}
+      INFRA_ACCESS_TOKEN: ${{ secrets.INFRA_ACCESS_TOKEN }}
+      ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: ${{secrets.AWS_DEFAULT_REGION}}
+      SECRETS_S3_BUCKET: ${{ secrets.SECRETS_S3_BUCKET }}
+      SLACK_NOTIFICATION_CI_RUNS: ${{ secrets.SLACK_NOTIFICATION_CI_RUNS }}
+      REPOSITORY_URL: ${{secrets.ACCOUNT_ID}}.dkr.ecr.${{secrets.AWS_DEFAULT_REGION}}.amazonaws.com
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+      - name: Configuration
+        env:
+          AWS_CLI_VERSION: 1.18.101
+          DOCKER_VERSION: 20.10.3-r0
+          ECR_CREDENTIAL_HELPER_VERSION: 0.5.0
+        run: |
+          apk update \
+          && apk add --no-cache curl bash coreutils skopeo jq py3-pip \
+          rsync git openssh openssh-client tzdata \
+          && ln -s /usr/share/zoneinfo/UTC /etc/localtime \
+          && apk add --no-cache docker==${DOCKER_VERSION} \
+          && pip3 install awscli==${AWS_CLI_VERSION} \
+          && curl -Lo /usr/local/bin/ecs-cli \
+          https://amazon-ecs-cli.s3.amazonaws.com/ecs-cli-linux-amd64-latest \
+          && chmod +x /usr/local/bin/ecs-cli \
+          && wget https://amazon-ecr-credential-helper-releases.s3.us-east-2.amazonaws.com/${ECR_CREDENTIAL_HELPER_VERSION}/linux-amd64/docker-credential-ecr-login \
+          && chmod +x docker-credential-ecr-login \
+          && mv docker-credential-ecr-login /bin/docker-credential-ecr-login \
+          && wget https://releases.hashicorp.com/terraform/1.1.7/terraform_1.1.7_linux_amd64.zip \
+          && unzip terraform_1.1.7_linux_amd64.zip && rm terraform_1.1.7_linux_amd64.zip \
+          && mv terraform /usr/bin/terraform
+      - name: Start deployment slack notification
+        run: |
+          curl -X POST -H 'Content-type: application/json' \
+          --data "{\"text\":\"The $SERVICE service(s) deployment on $ENV environment has begun.\n \"}" \
+          $SLACK_NOTIFICATION_CI_RUNS
+      - name: Set DOCKER_TAG
+        run: |
+          echo "DOCKER_TAG=${REPOSITORY_URL}/emrit-${ENV}-${SERVICE}:${BUILD_NUMBER}" >> $GITHUB_ENV
+      - name: Build image
+        run: docker build --compress -t ${DOCKER_TAG} --build-arg git_pat=${ACCESS_TOKEN} -f Dockerfile .
+      - name: docker login
+        run: docker login --username AWS --password "$(aws ecr get-login-password)" ${REPOSITORY_URL}
+      - name: push image
+        run: docker push ${DOCKER_TAG}
+      - name: Deploy to ECS
+        run: |
+          git clone --single-branch --branch ${INFRA_REPO_BRANCH} "https://${INFRA_ACCESS_TOKEN}@${INFRA_REPO}" infra
+          cd infra/env/${ENV}
+          aws s3 cp s3://${SECRETS_S3_BUCKET}/ . --recursive --exclude "*" --include "*.secret.auto.tfvars"
+          OLDIFS="$IFS"
+          IFS=","
```

Comment:
Referencing [search PR comment](https://github.com/emritio/search/pull/2#discussion_r882516771) pertaining to these lines

Reply:
Well remembered, thanks!

Reply:
These lines set the delimiting character when iterating through a string as in the for loop below. Normally I think it takes each character, but when you set this it means that it will take each value in a comma separated string eg "one, two" will have "one" and "two" instead of "o", "n", "e" etc